app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        name = self.name
+        health = self.health
+        hidden = self.hidden
+        return f"{{Name: {name}, Health: {health}, Hidden: {hidden}}}"
```

Comment:
no need to create these variables
you can do it like this
`return f"{{Name: {self.name}..."`